tests/unittests/test_formatting_dates.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+from unittest import mock
+import singer
+import tap_github.__init__ as tap_github
```

Comment:
it is not clear to me why do we need this weird import?

Reply:
That was not necessary. I updated it to be more readable.